x/incentives/keeper/gauge.go

```diff
@@ -295,69 +295,6 @@ func (k Keeper) getWeightBySplittingPolicy(ctx sdk.Context, poolId uint64, split
 	}
 }
 
-// AddToGaugeRewardsFromGauge transfer coins from a group gauge to its internal gauges.
-// Prior to calling this function, we make sure that the internalGaugeId is linked with the associated groupGaugeId.
-// Note: we donot have to bankSend for this gauge transfer because all the available incentive has already been bank sent
-// when we create Group Gauge. Now we are just allocating funds from groupGauge to internalGauge.
-// CONTRACT: internalGaugeId should be present in the group gauge.
-// TODO: revisit, clean up, and test this function as part of https://github.com/osmosis-labs/osmosis/issues/6402
-func (k Keeper) AddToGaugeRewardsFromGauge(ctx sdk.Context, groupGaugeId uint64, coins sdk.Coins, internalGaugeId uint64) error {
```

Comment:
Note to reviewer: this is unused, and I doubt we will need it

Reply:
Agree with removing